taskcluster/scripts/build/android_build_debug.sh

```diff
@@ -5,12 +5,8 @@
 set -e
 
 # This script is used in the Android Debug (universal) build task
-git submodule init
-git submodule update
-# glean
```

Comment:
@strseb please review again :)

Owen and I were discussing the upcoming ability to remove import_languages and that it's only used for qmake ios build and, maybe something to do with addons, but it's not needed for cmake builds.

So, I took the liberty of removing both generate_glean and import_languages from this and `android_build_release`.

Let me know if you object.